msbrainpy/atlas/query.py

```diff
@@ -142,3 +146,165 @@ def generateQueryRows(model, criteria, options='[only$eq''genes.entrez_id,data_s
         print('new start row: {}'.format(startRow))
         done = retrieved == 0
         yield queryResult
+
+
+
+# ----------------------------------------------- Get Gene Images ------------------------------------------------------
+def download_gene_images(entrez_id, directory_path=None, age_id=15, product_id=1, verbose=False):
+    download_from_section_data_set(get_section_images, entrez_id, directory_path=directory_path, age_id=age_id, 
+                                   product_id=product_id, verbose=verbose)
+
+
+def get_section_images(data_set_id, out_dir, verbose=False):
+    section_images = section_image_query(data_set_id, verbose=verbose)
+    download_section_images(section_images, out_dir, check_img_exists=True, return_img_n=None)
+
+
+# function for retrieving lists of images for a given data set
+def section_image_query(data_set_id, verbose=True):
+    url_0 = "http://api.brain-map.org/api/v2/data/query.json?criteria=model::SectionImage,rma::criteria,"
+    url_1 = '[data_set_id$eq{}]'.format(data_set_id)
+    url = url_0 + url_1
+    response = requests.get(url)
+    image_json = response.json()
+    if verbose:
+        print('{} images matching the criteria were found'.format(image_json['num_rows']))

```

Comment:
For this you can use a py3.6+ standard called f-strings:
```suggestion
        print(f'{image_json["num_rows"]} images matching the criteria were found')

```